### PR TITLE
feat(android-setup): detect-adb-step dialog implementation

### PR DESCRIPTION
**Pull request by JGibson2019**
#### Description of changes
Added the spinner for `detecting-adb` state per the Figma mock here: https://www.figma.com/file/Bugs8uNCHdCzOKzhq06nNK2o/Mobile-User-flow?node-id=1285%3A42078 

Image of how the dialog appears now with the changes from this PR:
image

#### Pull request checklist

- [N/A] Addresses an existing issue: #0000
- [X] Ran `yarn fastpass`
- [X] Added/updated relevant unit test(s) (and ran `yarn test`)
- [X] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [X] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [X] (UI changes only) Added screenshots/GIFs to description above
- [X] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/electron/views/device-connect-view/components/android-setup/detect-adb-step.tsx [hunk 1]
```diff
@@ -2,11 +2,25 @@
...
 
+import { androidSetupSpinner } from '../../device-connect-view.scss';
+
 export const DetectAdbStep = NamedFC<CommonAndroidSetupStepProps>(
     'DetectAdbStep',
     (props: CommonAndroidSetupStepProps) => {
-        return <>step: {props.androidSetupStoreData.currentStepId}</>;
+        const renderSpinner = () => {
+            return (
+                <Spinner
```
Comment (karanbirsingh) [inline src/electron/views/device-connect-view/components/android-setup/detect-adb-step.tsx:15]: It looks like this code creates a function on line 13 (`renderSpinner`) and then returns its result after calling it. We may be able to return the spinner directly, e.g. lines 14 - 22

File: src/electron/views/device-connect-view/device-connect-view.scss [hunk 2]
```diff
@@ -11,5 +11,8 @@
     .main-content-wrapper {
         flex: 1 1 auto;
         overflow: auto;
+        .android-setup-spinner {
```
Comment (dbjorge) [inline src/electron/views/device-connect-view/device-connect-view.scss:14]: Could this go in a CSS module specific to detect-adb-step?

Comment (JGibson2019) [reply src/electron/views/device-connect-view/device-connect-view.scss:14]: I plan on using it in other android-setup spinners, so what I'll do is create a standalone CSS module in the android-setup folder!

File: src/electron/views/device-connect-view/components/android-setup/detect-adb-step.tsx (hunk 1 above)
Comment (JGibson2019) [reply src/electron/views/device-connect-view/components/android-setup/detect-adb-step.tsx:15]: True; I'm not sure why I didn't just simply return the spinner. Thanks for noticing this

**Pull request merged**